plugins/UM3NetworkPrinting/src/Network/ZeroConfClient.py

```diff
@@ -152,6 +154,16 @@ def _onServiceAdded(self, zero_conf: Zeroconf, service_type: str, name: str) ->
 
         return True
 
+    @staticmethod
+    def _getAddress(info: ServiceInfo) -> str:
```

Comment:
Info address can be None as it seems 
```suggestion
    def _getAddress(info: ServiceInfo) -> Optional[str]:
```